snap_http/http.py

```diff
@@ -15,39 +16,49 @@ class SnapdHttpException(Exception):
     """An exception raised during HTTP communication with snapd."""
 
 
-def get(path: str) -> SnapdResponse:
+def get(
+    path: str,
+    query_params: Optional[Dict[str, Any]] = None,
```

Comment:
what do you think about using kwargs instead? Can still pass them to `_make_request` as `query_params=query_params`.
```python
**query_params: Any
```

Reply:
Switched to `**kwargs`